### PR TITLE
[16.0][ADD] stock_account_anglo_saxon_cogs_kit

### PR DESCRIPTION
**Pull request by MarinaAForgeFlow**
The Odoo Official documentation recommends to use consumable for kit products and only to be
storable in case Anglosaxon accounting:

https://www.odoo.com/documentation/16.0/applications/inventory_and_mrp/manufacturing/management/kit_shipping.html

"Although kits should almost always be set to Consumable,
companies using Anglo-Saxon accounting might need to create kits as a Storable Product. This is 
because when processing invoices for kits, the Cost of Goods Sold (COGS) will be posted in accounting journals."

However, that forces companies to track quantities for kits, and that is not always desirable. 

With this module companies using Anglo-Saxon accounting don't need to set the kit products to be
storable if they don'e want to track quantities for the kit.

cc @ForgeFlow

### CODE REVIEW

File: stock_account_anglo_saxon_cogs_kit/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Account Anglo Saxon COGS Kit",
+    "category": "Accounting",
+    "version": "16.0.1.1.0",
```
Comment (JordiMForgeFlow) [inline stock_account_anglo_saxon_cogs_kit/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

**Comment by AaronHForgeFlow** [comment]
LGTM! just squash commits into one please

File: stock_account_anglo_saxon_cogs_kit/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2023 ForgeFlow S.L.
```
Comment (JordiMForgeFlow) [inline stock_account_anglo_saxon_cogs_kit/__manifest__.py]:
```suggestion
# Copyright 2024 ForgeFlow S.L.
```

Also add the copyright to the account_move_line.py file. For the rest LGTM!

File: stock_account_anglo_saxon_cogs_kit/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Account Anglo Saxon COGS Kit",
+    "category": "Accounting",
+    "version": "16.0.1.0.0",
+    "depends": ["stock_account"],
+    "data": [],
+    "author": "ForgeFlow," "Odoo Community Association (OCA)",
```
Comment (MiquelRForgeFlow) [inline stock_account_anglo_saxon_cogs_kit/__manifest__.py]:
```suggestion
    "author": "ForgeFlow,Odoo Community Association (OCA)",
```

File: stock_account_anglo_saxon_cogs_kit/README.rst [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-base_vat_optional_vies
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/account-financial-tools&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module changes the anglo saxon COGS eligible method to allow kits to be eligible
```
Comment (MiquelRForgeFlow) [inline stock_account_anglo_saxon_cogs_kit/README.rst]: create readme fragments...

**Review by JordiMForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM!

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
This module depends on sale_mrp as the main need is to cover the case for kits consumable. Also added a test.

File: stock_account_anglo_saxon_cogs_kit/readme/DESCRIPTION.rst [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+https://www.odoo.com/documentation/16.0/applications/inventory_and_mrp/manufacturing/management/kit_shipping.html
+
+"Although kits should almost always be set to Consumable,
+companies using Anglo-Saxon accounting might need to create kits as a Storable Product. This is 
+because when processing invoices for kits, the Cost of Goods Sold (COGS) will be posted in accounting journals."
+
+However, that forces companies to track quantities for kits, and that is not always desirable. 
+
+With this module companies using Anglo-Saxon accounting don't need to set the kit products to be
+storable if they don'e want to track quantities for the kit.
```
Comment (JordiMForgeFlow) [inline stock_account_anglo_saxon_cogs_kit/readme/DESCRIPTION.rst]:
```suggestion
storable if they don't want to track quantities for the kit.
```

sorry for the nitpicking 😄

Comment (AaronHForgeFlow) [reply stock_account_anglo_saxon_cogs_kit/readme/DESCRIPTION.rst]: haha good catch!

**Pull request merged**